server/database.go

```diff
@@ -580,7 +580,7 @@ func (s *Server) registerDatabaseRoutes(g *echo.Group) {
 		}
 		if !s.licenseService.IsFeatureEnabled(api.FeatureReadReplicaConnection) {
 			// In the non-enterprise version, we should allow users to set HostOverride or PortOverride to the empty string.
```

Comment:
needs to update the comment.

Also the feature gate needs to be changed, otherwise, user can't change host, port

Reply:
Good catch!